### PR TITLE
Feature/dropdown button

### PR DESCRIPTION
**Pull request by Naseer-007**
- Created and Styled dropdown button
- Reused primary secondary and plain tokens for dropdown
- Added missing token variables in the selected.scss
- Added examples to the QA page for testing

### CODE REVIEW

File: component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts [hunk 1]
```diff
@@ -1,26 +1,61 @@
...
+  @Input() placeholderText? : string = '';
+  @Input() disabled?: boolean;
+  @Input() category?: keyof typeof ButtonCategories;
 
   showPlaceholder : boolean = false;
+  selected: boolean = false;
 
   constructor() { }
 
   ngOnInit(): void {
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts:39]: Remove ': void'

File: component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts [hunk 2]
```diff
@@ -1,26 +1,61 @@
...
-  @Input() placeholderText? : string;
+  @Input() label? : string = '';
+  @Input() placeholderText? : string = '';
+  @Input() disabled?: boolean;
+  @Input() category?: keyof typeof ButtonCategories;
 
   showPlaceholder : boolean = false;
+  selected: boolean = false;
 
   constructor() { }
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts:37]: Remove

**Review by kristopher-houston — APPROVED** [review]

File: component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts (hunk 2 above)
Comment (Naseer-007) [reply component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts]: removed.

File: component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts (hunk 1 above)
Comment (Naseer-007) [reply component-library/component-lib/src/lib/shared/dropdown/drop-down.component.ts]: removed.

**Pull request merged**